pkg/operation/seed/seed.go

```diff
@@ -488,83 +490,100 @@ func RunReconcileSeedFlow(
 		parsers                           = strings.Builder{}
 		fluentBitConfigurationsOverwrites = map[string]interface{}{}
 		lokiValues                        = map[string]interface{}{}
+		centralLokiStarage                = "100Gi"
 	)
 
 	lokiValues["enabled"] = loggingEnabled
 
-	// check if loki disabled in gardenlet config
-	if loggingConfig != nil &&
-		loggingConfig.Loki != nil &&
-		loggingConfig.Loki.Enabled != nil &&
-		!*loggingConfig.Loki.Enabled {
-		lokiValues["enabled"] = false
-		if err := common.DeleteLoki(ctx, seedClient, gardenNamespace.Name); err != nil {
-			return err
-		}
-	}
-
 	if loggingEnabled {
-		lokiValues["authEnabled"] = false
-
 		lokiVpa := &autoscalingv1beta2.VerticalPodAutoscaler{ObjectMeta: metav1.ObjectMeta{Name: "loki-vpa", Namespace: v1beta1constants.GardenNamespace}}
 		if err := seedClient.Delete(ctx, lokiVpa); client.IgnoreNotFound(err) != nil && !meta.IsNoMatchError(err) {
```

Comment:
Btw, unrelated to the PR, but why do we delete the VPA here?

Reply:
I don't know. Maybe it is leftover. We don't have Loki VPA